WireDomain/Sources/WireDomain/Repositories/Message/MessageLocalStore.swift

```diff
@@ -1021,4 +1021,29 @@ public final class MessageLocalStore: MessageLocalStoreProtocol {
         }
     }
 
+    // MARK: - Backup / Restore
+
+    public func totalMessageCountForBackup() async throws -> Int {
```

Comment:
Since you have a new specific store for backups, can we revert these changes?

Reply:
done